Try to load libpcre.so.1 if libpcre.so.0 isn't available.

2012-03-11 Martin von Gagern

References:
https://bugs.gentoo.org/404195

Index: parrot-4.1.0/runtime/parrot/library/pcre.pir
===================================================================
--- parrot-4.1.0.orig/runtime/parrot/library/pcre.pir
+++ parrot-4.1.0/runtime/parrot/library/pcre.pir
@@ -78,6 +78,11 @@ LIB_DEFAULT:
     loaded = defined libpcre
     if loaded goto LIB_LOADED
 
+    # See https://bugs.gentoo.org/404195
+    loadlib libpcre, 'libpcre.so.1'
+    loaded = defined libpcre
+    if loaded goto LIB_LOADED
+
     branch LIB_FAILED
 
 LIB_WIN32:
